app/models/custom_filter.rb

```diff
@@ -35,6 +35,13 @@ class CustomFilter < ApplicationRecord
   before_validation :clean_up_contexts
   after_commit :remove_cache
 
+  def expires_in
+    return @expires_in if defined?(@expires_in)
```

Comment:
I realize this PR is 5 years old, but bumped into this while doing something else ... it seems like this memoization is never actually used? (ie, this ivar is never assigned to anything) or am I missing something here?

Reply:
I don't think you're missing anything, I think this line is useless.